core/src/main/scala/matryoshka/instances/fixedpoint/package.scala

```diff
@@ -67,4 +67,18 @@ package object fixedpoint {
         case (_, _)                 => NilF()
       }
   }
+
+  /** Encodes a function that may diverge.
+    */
+  type Partial[A] = Nu[A \/ ?]
+  type ⊥[A] = Partial[A]
```

Comment:
Is this type really the bottom type though? I think `Partial` is a good name... But `_|_` typically refers to a type that has no canonical elements (e.g. whose elements always diverge), not to a type whose elements *may* diverge. (But it's possible it's been used this way–I hadn't seen it though)